lib/plug/csrf_protection.ex

```diff
@@ -103,17 +103,34 @@ defmodule Plug.CSRFProtection do
 
   def init(opts), do: opts
 
-  def call(conn, _opts) do
+  def call(conn, opts) do
+    opts = verify_opts opts
```

Comment:
Wouldn't it better to get this error during compilation and not to run server with incorrect one?